docs/guides/gradle.md

```diff
@@ -142,7 +142,7 @@ language and responsibility boundaries.
 
 If your server should be deployed as a whole, use a single `web-server` for all the contexts. If you
 would like to deploy different contexts separately, declare a specific `web-server` subprojects
-for each of those contexts. See the [API reference]({{site.core_api_doc}}/server/io/spine/server/ServerEnvironment.html#use-io.spine.server.transport.TransportFactory-io.spine.base.EnvironmentType-)
```

Comment:
It looks like this link should point to the specific section or some particular method in the `ServerEnvironment` documentation, but it does not. I am not sure about my replacement, so PTAL @armiol.